lib/tasks/h2o.rake

```diff
@@ -59,41 +59,4 @@ namespace :h2o do
     end
     puts 'Done!'
   end
-
```

Comment:
Starting with a task and figuring out what to do about the output (if it proves meaningful) sounds good to me. When I mentioned to Casey that there's a more involved migration one could do with the versioning, what I had in mind was applying HTMLUtils by EFFECTIVE_DATE as passed from each annotation, so if a document had 3 annotations applied across two years, it would calculate the proper offsets for first annotation using V1, the second using V2, and the third using V3 (or whatever versions lined up with their created_at). Finding a sane way to test whether that would be worthwhile would be... worthwhile.

Reply:
Yeah, I betcha that isn't worth it.

So I started with the manual approach, looking at what actually changed between v1-v6, tracking down cases that contained relevant markup, filtering to find only those that are public and annotated. Punting on the nokogiri vs nokogumbo change for the time being, it looks like only the move to stop unwrapping `div`s between v3 and v4 might be significant, for a small handful of cases.

This case contains 103 `div` tags, an extreme outlier. Check out https://h2o-experiment.lil.tools/casebooks/25332-criminal-law-reading-group-2016/resources/5.1.2.10-prosecutor-v-kunarac#/ around paragraph 34, compared with production, https://opencasebook.org/casebooks/25332-criminal-law-reading-group-2016/resources/5.1.2.10-prosecutor-v-kunarac#/

I can't quite tell what I'm seeing: lots of overlapping offsets on h2o-experiment, I think? The divs seem like a likely culprit... (Note, the annotations are shifted in prod, for as-yet-unknown reasons (I have a couple of theories), but they aren't overlapping.)